packages/card/src/__demo__/examples/card-title/withCategoryTitle.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import {
+  createStyledComponent,
+  mineralTheme
+} from '@mineral-ui/component-utils';
+import Card from '../../../Card';
+import CardBlock from '../../../CardBlock';
+import CardTitle from '../../../CardTitle';
+
+const DemoLayout = createStyledComponent('div', {
+  '& > *': {
+    width: '33.333%'
+  }
+});
+
+export default {
+  title: 'With a category title',
+  backgroundColor: mineralTheme.color_gray_10,
+  description:
+    'If you provide a category title, it will display above the title, which will automatically display in the minor style.',
+  scope: { Card, CardBlock, CardTitle, DemoLayout },
+  source: `<DemoLayout>
+    <Card>
```

Comment:
i never did, but given the number of times i've caught indentation issues, maybe i should... do we want a library to do our indentation for us at runtime or do we want to have it correct in the code too?

i could also put in a console warning if the source has a newlines or tabs in the wrong places